portfolio/src/main/webapp/comments.html

```diff
@@ -19,11 +19,14 @@
         <form id = "commentForm">
             <textarea name="comment" rows="5" cols="60" type="text" placeholder="Insert your comment here"></textarea>
             <br/>
-            <button type="submit" onclick="submitForm()">Submit</button>
+            <button class = "btn btn-blue" type="submit" onclick="submitForm()">Submit</button>
         </form>
-        <div class="separator">Comments</div>
-        <label for="quantity">How many comments would you like to see? (1 to 20)</label>
-        <input type="number" id="quantity" name="quantity" min="1" max="20" value="10" onchange="callFetch(value)">
+        <a href = "" class = "btn btn-white" id="loginRoute"> Login </a>
+        <div class="subheading">
+            <div class="comments-separator">Comments</div>
+            <label for="quantity" class="labeltag">Number of comments:(1 to 20)</label>
+            <input type="number" id="quantity" name="quantity" min="1" max="20" value="10" onchange="callFetch(value)">
```

Comment:
If the user logout and change the value here, will they receive comments?

Reply:
Yes they will. The user can still see comments and decide how many comments to see, they just can post a new comment or delete comments

Reply:
IC, Thanks for the clarification.